### ISSUE TITLE
Viewing logs in omnia admin - random order

### ISSUE THREAD

**Issue by PetterT2** [preciofishbone/Omnia-Foundation#43]
Logs in omnia admin is displayed in random order if timespan between log entries is to short, i.e. it's hard to follow  the flow of events.

**Comment by andtii** [comment]
Timestamp is not good enough, the orderby should be using row_number

**Comment by khdi** [comment]
After some testing it seems like the order is still correct as long the CreatedAt is not exactly the same. 
![image](https://user-images.githubusercontent.com/20087013/29697849-d3c68448-897b-11e7-96e7-437958711e71.png)
